packages/containerd-2.0/containerd-2.0.spec

```
@@ -0,0 +1,175 @@
+%global goproject github.com/containerd
+%global gorepo containerd
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 2.0.5
+%global rpmver %{gover}
+%global gitrev fb4c30d4ede3531652d86197bf3fc9515e5276d9
+
+%global _dwz_low_mem_die_limit 0
+
+Name: %{_cross_os}%{gorepo}-2.0
+Version: %{rpmver}
+Release: 1%{?dist}
+Epoch: 1
+Summary: An industry-standard container runtime
+License: Apache-2.0
+URL: https://%{goimport}
+Source0: https://%{goimport}/archive/v%{gover}/%{gorepo}-%{gover}.tar.gz
+Source1: containerd.service
+Source2: containerd-config-toml_k8s_containerd_sock
+Source3: containerd-config-toml_basic
+Source4: containerd-config-toml_k8s_nvidia_containerd_sock
+Source5: containerd-tmpfiles.conf
+Source6: containerd-cri-base-json
+
+# Mount for writing containerd configuration
+Source100: etc-containerd.mount
+
+# Create container storage mount point.
+Source110: prepare-var-lib-containerd.service
+
+# Drop-ins to disable igzip or pigz if the other implementation is preferred.
+Source200: containerd-disable-igzip.conf
+Source201: containerd-disable-pigz.conf
+
+Source1000: clarify.toml
+
+# Patches to support moving from containerd-1.7 to 2.0
+Patch1001: 1001-Revert-Don-t-allow-io_uring-related-syscalls-in-the-.patch
+
+BuildRequires: git
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}runc
+Requires: %{_cross_os}containerd(optimized-gunzip)
```

This is incorrect; you don't want to take the dependency from a different version of containerd:
```suggestion
Requires: %{name}(optimized-gunzip)
```